tests/cli/test_check.py

```diff
@@ -181,3 +181,43 @@ def test_check_access(context: Context, rc, cli: SpintaCliRunner, tmp_path):
     assert result.exit_code != 0
     assert result.exc_info[0] is InvalidValue
     assert result.exception.context.get('given') == "test"
+
+
+def test_check_namespace_two_manifests(context: Context, rc, cli: SpintaCliRunner, tmp_path):
```

Comment:
Maybe the naming could be changed? It seems like these are not two manifests files, but rather ns goes between defined datasets, when it should be atop of them